content/en/blog/2024/ambient-reaches-beta/index.md

```diff
@@ -0,0 +1,111 @@
+---
+title: "Istio’s new sidecar-less ‘Ambient Mode’ reaches Beta in version 1.22"
```

Comment:
Should i use double quote here? @dhawton

Reply:
Ok cool, thanks! Understood the timing.

Reply:
finally at the PC and it's merged already.  I have thoughts 😄 

"Ambient Mode" shouldn't be capitalised, it's not a brand, it shouldn't be in quotes, but should be emphasised more than "sidecar-less" (which probably shouldn't have a hyphen)

I'll go back to my hill...